controllers/src/main/scala/cromwell/pipeline/controller/ProjectFileController.scala

```diff
@@ -35,6 +37,17 @@ class ProjectFileController(wdlService: ProjectFileService)(implicit val executi
             }
           }
         }
+      },
+      path("files" / "configurations") {
+        post {
+          entity(as[ProjectBuildConfigurationRequest]) { request =>
+            onComplete(wdlService.buildConfiguration(request.projectId, request.projectFile)) {
+              case Success(Left(e))              => complete(StatusCodes.ImATeapot, e.errors)
```

Comment:
Funny, but how about proper error code? :)

Reply:
Decided to return 422 status code instead, thx